docs/Options.md

```diff
@@ -141,11 +141,18 @@ Type: `Number`<br>
 Default: `0` (don't bail, run all tests)
 
 ### specFileRetries
-The number of times to retry an entire specfile when it fails as a whole
+The number of times to retry an entire specfile when it fails as a whole.
 
 Type: `Number`<br>
 Default: `0`
 
+### specFileRequeue
+Where retried specfiles should be reinserted into the queue.
```

Comment:
It's still not clear what behavior will be. Could you explain a little bit more. You wrote `where` but it's unclear how it will work